### ISSUE TITLE
Cannot obtain accuracy stated in the doc for inception_v3 pretrained on Imagenet

### ISSUE THREAD

**Issue by m-parchami** [pytorch/hub#290]
Hi.
I'm trying to evaluate `inception_v3` pretrained model from the hub on Imagenet (ILSVRC 2012) test set. I use the following evaluation code:
```Python
def compute_accuracy(self):
        num_correct = 0
        num_images = 0

        _IMAGE_MEAN_VALUE = [0.485, 0.456, 0.406]
        _IMAGE_STD_VALUE = [0.229, 0.224, 0.225]
        imgnet_loader = torch.utils.data.DataLoader(
            ImageFolder('/home/amin/dataset/ILSVRC/val', 
            transforms.Compose([
                transforms.Resize(299),
                transforms.CenterCrop(299),
                transforms.ToTensor(),
                transforms.Normalize(mean=_IMAGE_MEAN_VALUE, std=_IMAGE_STD_VALUE),
            ])),
            batch_size=16, shuffle=True,
            num_workers=8, pin_memory=True)
        self.model = torch.hub.load('pytorch/vision:v0.10.0', 'inception_v3', pretrained=True).cuda()
        self.model.eval()

        for i, (images, targets) in \
            enumerate(tqdm(imgnet_loader,  desc="Compute Accuracy", total=len(loader))):
            images = images.cuda() if torch.cuda.is_available() else images.cpu()
            targets = targets.cuda() if torch.cuda.is_available() else targets.cpu()
            output_dict = self.model(images)
            pred=output_dict.argmax(dim=1)
            
            num_correct += (pred == targets).sum().item()
            num_images += images.size(0)

        classification_acc = num_correct / float(num_images) * 100
        return classification_acc
```

However, the accuracy I get is `77.216` while it should be `77.45` according to [this page](https://pytorch.org/hub/pytorch_vision_inception_v3/). I figured that the model has a transform_input as preprocess in itself. So if we are doing normalization beforehand (as suggested in the [example code](https://pytorch.org/hub/pytorch_vision_inception_v3/)), we should set `transform_input` to false. So if I add `self.model.transform_input = False`, I get `77.472`, Which is closer to the expected value but not exactly the same.
I appreciate any input on this.

Thanks.

**Comment by NicolasHug** [comment]
@m-parchami I'd recommend using the torchvision training references to check the accuracy of a given model - that's what they use to report accuracies - https://github.com/pytorch/vision/tree/main/references/classification#inception-v3
Also please note that this isn't a tochhub-related issue, more a torchvision issue (you're in luck, we maintain both).

Various things can affect the results. Obviously slight differences in the code, but also batch size, shuffling, some samples potentially being dropped or duplicated depending on the number of GPUs, the use of CUDA determinism, etc...
